public/blog/first-ever-truffle-retreat.md

```diff
@@ -0,0 +1,123 @@
+# Our First Ever Truffle Retreat
+
+![Truffle](/blog/images/first-ever-truffle-retreat/medium-truffle-retreat.jpg)
+
+The Truffle team recently got together late September to define our roles, our mission statement, and our product roadmap - and had some fun in the process.  
+
+----------
+
+The Truffle team is fully remote. This has a lot of advantages in that it allows for a wide range of people available to work on your project; but it can also make it difficult to get to know your coworkers. 
+
+I started Truffle in early 2015 as the sole developer, and it stayed that way for nearly a year and half. John McDowell joined in late 2016, and by the time of the retreat we happily brought Nick D’Andrea and Josh Quintal on board. All interviews happened over Skype.
+
+I live in Yakima, Washington. John lives in Victoria BC. Nick lives in Philadelphia, and Josh lives in Detroit. That’s a lot of distance between us. 
```

Comment:
Suggest "Victoria, British Columbia", for those who don't know their Canadian abbreviations.

Reply:
✔️